composer.json

```diff
@@ -14,28 +14,17 @@
     },
     "require": {
         "php": ">=5.4.4",
-        "symfony/symfony": "~2.6",
         "twig/extensions": "~1.0",
         "symfony/assetic-bundle": "~2.3",
         "symfony/swiftmailer-bundle": "~2.3",
         "symfony/monolog-bundle": "~2.4",
-        "sensio/distribution-bundle": "~2.3|~3.0",
-        "sensio/generator-bundle": "~2.3",
         "incenteev/composer-parameter-handler": "~2.0",
-        "tedivm/stash-bundle": "0.4.*",
         "ezsystems/ezpublish-kernel": "dev-master",
         "ezsystems/ezpublish-legacy": "dev-master",
         "ezsystems/platform-ui-bundle": "dev-master",
         "ezsystems/platform-ui-assets-bundle": "~0.1",
         "ezsystems/demobundle": "dev-master",
-        "ezsystems/comments-bundle": "dev-master",
```

Comment:
This one is here because we explicitly activate it in EzPublishKernel. So please leave it, even if it's present in DemoBundle.